reverse_engineering/cassandraHelper.js

```diff
@@ -922,7 +923,7 @@ module.exports = (_) => {
 			documents: data.records,
 		};
 	
-		if (data.table.columns && data.table.columns.length) {
+		if (data?.table?.columns?.length) {
```

Comment:
not critical, but personally I always check if the root variable is being _nullish_ checked (`data?.`) previously as well. But if, for example, the previous accessing to that variable occur without similar check (like few lines above) - then it looks a bit inconsistent 🙂